scalableminds/webknossos
# Improve performance of Bounding Box Context Menus

### URL of deployed dev instance (used for testing):
- https://bboxcontextmenuperf.webknossos.xyz/

### Steps to test:
- Go to bounding box tab within an annotation, have some bounding boxes laying around
- within the bounding box tab, left and right clicking the three dots aswell as right-clicking a bbox entry (almost) anywhere should open the context menu
- all enabled actions should be doing something and closing the context menu afterwards
- the context menu should close when clicking somewhere within the bounding box tab
- make sure the skeleton tab and the context menus there are looking good and behaving appropriately (minor change there to prevent horizontal scrollbar from showing up)

### TODOs:
- [x] right-clicking the whole bbox list entry should open the context menu (find out how to pass menu there)
- [x] left-clicking the three dots should open the context menu
- [x] close context menu after action
- [x] fix gitch and horizontal scroll bar when opening context menu

------
(Please delete unneeded items, merge only when none are left open)
